drivers/hmis/app/graphql/mutations/assign_staff.rb

```diff
@@ -0,0 +1,45 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class AssignStaff < CleanBaseMutation
+    argument :household_id, ID, required: true
+    argument :assignment_type_id, ID, required: true
+    argument :user_id, ID, required: true
+
+    field :staff_assignment, Types::HmisSchema::StaffAssignment, null: true
+
+    def resolve(household_id:, assignment_type_id:, user_id:)
+      household = Hmis::Hud::Household.preload(:project).find_by(household_id: household_id, data_source_id: current_user.hmis_data_source_id)
+      raise 'Not found' unless household
+
+      access_denied! unless current_user.permissions_for?(household.project, :can_edit_enrollments)
+
+      raise 'Staff Assignment not enabled' unless household.project.staff_assignments_enabled?
+
+      assignment_type = Hmis::StaffAssignmentType.find(assignment_type_id)
+      user = Hmis::User.find(user_id)
```

Comment:
It seems like it should mirror what we have in the picklist?
```suggestion
      user = Hmis::User.can_edit_enrollments_for(household.project).find(user_id)
```